spec/rails_helper.rb

```diff
@@ -60,4 +61,7 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+
+  # FactoryGirl Config
+  config.include FactoryGirl::Syntax::Methods
```

Comment:
@hiattp this one is already in https://github.com/lockstep/rails_new/blob/master/spec/support/factory_girl.rb

Reply:
Oh, thats weird. When I created a new app it complained that create wasn't defined (I had to reference fg explicitly). Have you confirmed whether this works?

Reply:
It does, as long as you `require 'rails_helper'` in your spec

Reply:
DOH! Always forget that line...

Reply:
@hiattp I did the exact same thing last week 😄